### PR TITLE
Fix clear cache docs release.

### PR DESCRIPTION
**Pull request by jessfraz**
Clear the cache for all files. Don't worry about what changed.

Move the docs release script back to the docker/docker repo, so we all can
easily watch the changes.

ping @tianon @tiborvass @SvenDowideit @moxiegirl

### CODE REVIEW

File: docs/release.sh [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+	if [ "" == "$DISTRIBUTION_ID" ]; then
+		echo "Skipping Cloudfront cache invalidation"
+		return
+	fi
+
+	dst=$1
+
+	aws configure set preview.cloudfront true
+
+	files=$(find site/ -not -name "*.md*" -type f | sed 's/site\///g')
```
Comment (jfrazelle) [inline docs/release.sh]: if you are wondering where the change is, its this line

**Comment by tiborvass** [comment]
@jfrazelle @SvenDowideit where did the `release.sh` come from previously? I see you added a new one, but the makefile already was running docs/release.sh.

**Comment by jfrazelle** [comment]
it was in the docker/docs-base repo

**Comment by jfrazelle** [comment]
I think it should be here

**Comment by tiborvass** [comment]
@jfrazelle oh right thanks

**Comment by tianon** [comment]
LGTM

**Comment by moxiegirl** [comment]
LGTM

File: docs/release.sh [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+	sh batchfile
+	echo "-----"
+}
+
+
+if [ "$OPTIONS" != "--dryrun" ]; then
+	setup_s3
+fi
+
+# Default to only building the version specific docs so we don't clober the latest by accident with old versions
```
Comment (tiborvass) [inline docs/release.sh]: @SvenDowideit @moxiegirl i think I must be tired but can you explain this? (I know this is not from this PR)

**Comment by jfrazelle** [comment]
where we at with this one :D

File: docs/release.sh (hunk 2 above)
Comment (tiborvass) [inline docs/release.sh]: Ping @SvenDowideit

**Comment by jfrazelle** [comment]
@SvenDowideit we need this before the release

File: docs/release.sh (hunk 2 above)
Comment (SvenDowideit) [inline docs/release.sh]: this was coded to allow us to publish an update to a previous version of the docs (which I've done a few times)

File: docs/release.sh (hunk 1 above)
Comment (SvenDowideit) [inline docs/release.sh]: and what a line it is :D

**Comment by SvenDowideit** [comment]
yup, you're right, this is the release script for the docker project atm (when I moved it there was still discussion about having other project docs elsewhere)

LGTM

**Comment by tiborvass** [comment]
LGTM

aka #wewillseehowitgoes

**Pull request merged**